dashboard/app/models/pd/application/teacher2021_application.rb

```diff
@@ -704,6 +704,8 @@ def to_csv_row(course)
           if school_stats
             if [:title_i_status, :students_total, :urm_percent].include? k
               row.push(school_stats[k] || school_stats.try(k) || "")
+            elsif k === :rural_status
```

Comment:
Do we need triple equals `===` here?

Reply:
probably not, my javascript and ruby got mixed up :)